### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by rushsteve1**
A couple of quick (and admittedly a bit dirty) changes to make things a bit clearer for the user and the developers:

1. Changed any instance of 'ownCloud' to 'nextcloud'. This updated a lot of links to the proper locations, and generally made it clearer that this is Nextcloud notes.

2. This one is more my opinion, but the 'New Note' text should be visible while not hovered over. The pop-in was ugly and jarring and it is much clearer to the user the function of that link.

Any further alterations you would like me to make please just ask.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,19 +11,19 @@ If you have questions about how to install or use ownCloud, please direct these
...
-* This repository ([notes](https://github.com/owncloud/notes/issues)) is *only* for issues within the ownCloud notes code.
-* __SECURITY__: Report any potential security bug to security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* This repository ([notes](https://github.com/nextcloud/notes/issues)) is *only* for issues within the nextcloud notes code.
+* __SECURITY__: Report any potential security bug to security@nextcloud.com following our [security policy](https://nextcloud.com/security/) instead of filing an issue in our bug tracker
 * Report the issue using our [template][template], it includes all the information we need to track down the issue.
 
 Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
 
-[template]: https://raw.github.com/owncloud/core/master/issue_template.md
+[template]: https://raw.github.com/nextcloud/core/master/issue_template.md
```
Comment (phsc) [inline CONTRIBUTING.md]: should be .../nextcloud/server/...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,19 +11,19 @@ If you have questions about how to install or use ownCloud, please direct these
...
-[forum]: https://forum.owncloud.org/
-[irc]: https://webchat.freenode.net/?channels=owncloud&uio=d4
-[irc-dev]: https://webchat.freenode.net/?channels=owncloud-dev&uio=d4
+[forum]: https://help.nextcloud.com/
+[irc]: https://webchat.freenode.net/?channels=nextcloud&uio=d4
+[irc-dev]: https://webchat.freenode.net/?channels=nextcloud-dev&uio=d4
 
 ### Contribute Code and translations
-Please check [core's contribution guidelines](https://github.com/owncloud/core/blob/master/CONTRIBUTING.md) for further information about contributing code and translations.
+Please check [core's contribution guidelines](https://github.com/nextcloud/core/blob/master/CONTRIBUTING.md) for further information about contributing code and translations.
```
Comment (phsc) [inline CONTRIBUTING.md]: should be .../nextcloud/server/...

**Comment by rushsteve1** [comment]
@phsc you're right, I updated it to be uppercased in the header comments.

File: css/notes.css [hunk 3]
```diff
@@ -9,7 +9,6 @@
 }
 
 #app-navigation #note-add span {
-    display: none;
```
Comment (Henni) [inline css/notes.css]: @jancborchardt what do you think about this?
We get the demand to show the text every once in a while.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -11,19 +11,19 @@ If you have questions about how to install or use ownCloud, please direct these
   - Go to one of the repositories, click "issues" and type any word in the top search/command bar.
   - You can also filter by appending e. g. "state:open" to the search string.
   - More info on [search syntax within github](https://help.github.com/articles/searching-issues)
-* This repository ([notes](https://github.com/owncloud/notes/issues)) is *only* for issues within the ownCloud notes code.
-* __SECURITY__: Report any potential security bug to security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* This repository ([notes](https://github.com/nextcloud/notes/issues)) is *only* for issues within the nextcloud notes code.
+* __SECURITY__: Report any potential security bug to security@nextcloud.com following our [security policy](https://nextcloud.com/security/) instead of filing an issue in our bug tracker
```
Comment (Henni) [inline CONTRIBUTING.md]: I'm not sure this advice is still valid or if that mail address even exists.
@LukasReschke any advice?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ## Submitting issues
 
-If you have questions about how to install or use ownCloud, please direct these to the [mailing list][mailinglist] or our [forum][forum]. We are also available on [IRC][irc].
+If you have questions about how to install or use nextcloud, please direct these to the [mailing list][mailinglist] or our [forum][forum]. We are also available on [IRC][irc].
```
Comment (Henni) [inline CONTRIBUTING.md]: It's nit-picking, but could you please uppercase `Nextcloud`?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -11,19 +11,19 @@ If you have questions about how to install or use ownCloud, please direct these
   - Go to one of the repositories, click "issues" and type any word in the top search/command bar.
   - You can also filter by appending e. g. "state:open" to the search string.
   - More info on [search syntax within github](https://help.github.com/articles/searching-issues)
-* This repository ([notes](https://github.com/owncloud/notes/issues)) is *only* for issues within the ownCloud notes code.
-* __SECURITY__: Report any potential security bug to security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* This repository ([notes](https://github.com/nextcloud/notes/issues)) is *only* for issues within the nextcloud notes code.
```
Comment (Henni) [inline CONTRIBUTING.md]: Here as well

File: css/notes.css (hunk 3 above)
Comment (korelstar) [inline css/notes.css]: I vote for displaying the text always. And I vote for displaying it more like a button, like in the contacts app (cp. https://github.com/nextcloud/notes/issues/14#issuecomment-256340853):
neuer-kontakt

File: CONTRIBUTING.md (hunk 4 above)
Comment (rushsteve1) [inline CONTRIBUTING.md]: I was also unsure if that email existed, so decided to change it anyway. If anyone knows better please just tell me.

File: css/notes.css (hunk 3 above)
Comment (rushsteve1) [inline css/notes.css]: Personally I think it adds clarity, and overall looks better. Having it look like a button like @korelstar said would also be nice, though was somewhat out of scope of what i was going for in this PR.

**Comment by rushsteve1** [comment]
@Henni I made the requested changes. Anything else you would like me to do?

**Comment by rushsteve1** [comment]
@Henni or @phsc Sorry to be annoying, but no one has responded for 4 days. And this is just a typo PR.

**Comment by jancborchardt** [comment]
Good stuff @rushsteve1! :) Added you to the Nextcloud organization by the way – welcome!

**Pull request merged**